tests/CppUTest/TestHarness_cTest.cpp

```diff
@@ -96,7 +96,7 @@ TEST(TestHarness_c, checkBool)
     CHECK_EQUAL_C_BOOL(1, 2);
     fixture->setTestFunction(_failBoolMethod);
     fixture->runAllTests();
-    fixture->assertPrintContains("expected <true>\n	but was  <false>");
```

Comment:
Hard tabs in strings replaced with \t so that file can be retabbed, if necessary, without damaging these strings.